src/pages/benchmarks/[benchmark_id]/index.tsx

```diff
@@ -0,0 +1,220 @@
+import React, { useEffect, useState } from "react";
+import Counter from "@/components/para_trials/counter";
+import $box from "@/styles/Box.module.css";
+import $button from "@/styles/Button.module.css";
+import { trpc } from "@/client/lib/trpc";
+import { useRouter } from "next/router";
+import ChevronLeftIcon from "@mui/icons-material/ChevronLeft";
+import ChevronRightIcon from "@mui/icons-material/ChevronRight";
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import ParaNav from "@/components/ParaNav";
+import Link from "next/link";
+import { useDebounce } from "react-use";
+
+interface dataUpdate {
```

Comment:
convention is for interface/type names to be PascalCase

```suggestion
interface DataUpdate {
```